docker/server/src/main/docker/Dockerfile

```diff
@@ -29,13 +29,17 @@ RUN set -eux; \
     rm -r /deephaven2-wheel
 
 ARG DEEPHAVEN_VERSION
-
 ADD serverApplicationDist/server-${DEEPHAVEN_VERSION}.tar /opt/deephaven
 
 RUN set -eux; \
     ln -s /opt/deephaven/server-${DEEPHAVEN_VERSION} /opt/deephaven/server
 
-COPY image-bootstrap.properties image-bootstrap.properties
+COPY configure/ /configure
+RUN set -eux; \
```

Comment:
as discussed, it makes more sense to me to either have a common startup script that passes this to the jvm, or have the jvm call out to python and ask for this, so that docker and local jvm behave the same

Reply:
as discussed on call, lets just file an issue and TODO this for later, we have a working option, and we think we can make it magically work at runtime.